### ISSUE TITLE
How did you start editing podcasts?

### ISSUE THREAD

**Issue by thexavismith** [iChris/ama#1]

**Comment by iChris** [comment]
I'm guessing you mean "...editing podcasts for money" as opposed to just editing my own podcasts? 

If it's the former: I got a message from Dave at [ShopTalk Show](http://shoptalkshow.com) about needing a new editor and wondering if I'd send an estimate. Which I did. And then Chris, also from ShopTalk Show, wanted an editor for his other podcast, [CodePen Radio](https://blog.codepen.io/radio/), and we talked business and worked out an arrangement for that show. And then a different Chris from [The Non-Breaking Space Show](http://goodstuff.fm/nbsp) wanted in on the editing action and so we got our lawyers together and hammered out a deal.

Basically if you want to hire me, you just need to either be named Chris or know someone named Chris.
